### PR TITLE
Added pulsating effect for icons. Use .fa-pulsate class.

### PR DESCRIPTION
**Pull request by dhruvparmar372**
Added a pulsating effect for icons. Use .fa-pulsate class to give effect.
Example has been added for the effect. 
Browsers Tested On - Chrome(Version 32.0.1700.77) and Firefox(26.0)

### CODE REVIEW

File: src/_includes/examples/pulsating.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="col-md-3 col-sm-4">
+      <p>
+        <button class="btn btn-default btn-lg"><i class="fa fa-heart fa-pulsate"></i></button>
+        <button class="btn btn-default btn-lg"><i class="fa fa-plus-circle fa-pulsate"></i></button>
+        <button class="btn btn-default btn-lg"><i class="fa fa-html5 fa-pulsate"></i></button>
+      </p>
+    </div>
+    <div class="col-md-9 col-sm-8">
+      <p>
+        Use the <code>fa-pulsate</code> class to get any icon to rotate. Works well with <code>fa-heart</code>
```
Comment (futhr) [inline src/_includes/examples/pulsating.html]: Nice work :+1:  ... think it should say "pulsate" here instead of "rotate".

**Comment by anaibol** [comment]
+1!

**Comment by lichtamberg** [comment]
Here are also some nice font awesome animations... Could maybe included? 

http://l-lin.github.io/font-awesome-animation/

**Comment by RaXoR** [comment]
:+1:

**Comment by markus-hsk** [comment]
+1

**Comment by robmadole** [comment]
First off, thanks for spending the time to post this PR. It's sat in the Font Awesome GitHub repo for far too long and we should have responded or addressed the work that you put into this.

Over the past year we've been hard at work on Font Awesome 5. It's now released and this PR no longer applies. So we're going to close it (with an apology for being so tardy).

If you think the work from this still applies feel free to reply and we'll take a second look.

– FA team

**Pull request closed without merging**